ci/gitlab/.gitlab-ci.yml

```diff
@@ -408,3 +408,18 @@ trigger-docsite-build:
   trigger:
     project: olp/edge/ota/documentation/ota-connect-docs
     branch: master
+
+osx-build:
+  stage: docker
+  before_script:
+    - brew unlink aktualizr-dev
+    - brew uninstall -f aktualizr-dev.rb
+  script:
+    - brew tap advancedtelematic/otaconnect
+    - brew install --HEAD -v aktualizr-dev
+    - brew link -f aktualizr-dev
+    - aktualizr --version
+  tags:
+    - osx
+  only:
+    - schedules
```

Comment:
We have another weekly schedule for Veracode scans, we probably don't need to run the osx-build in that case.

Using `only: variables: - $OSX_BUILD` and defining the variable in the schedule would solve that.

Also, it makes it easier to trigger the job manually without having to go through a schedule.

Reply:
The scheduling mechanism allows triggering a job manually too.
Anyway, I agree with you, it makes sense